testsuite/features/core/srv_create_activationkey.feature

```diff
@@ -24,28 +23,32 @@ Feature: Create activation keys
     And I should see a "Groups" link
     And I should see a "Activated Systems" link
 
+@rhlike_minion
+  Scenario: Create an activation key for RedHat-like minion
+    When I follow the left menu "Systems > Activation Keys"
+    And I follow "Create Key"
+    And I enter "RedHat like Test Key" as "description"
+    And I enter "RH-LIKE-KEY" as "key"
+    And I select "Test-Channel-x86_64" from "selectedBaseChannel"
```

Comment:
Why not `Fake-` here ? Or do I miss something?

Reply:
Ah right I should create a new channel for RedHat-like minions instead of reusing this one, it does not make sense to keep this channel here, nice catch.